ansible/roles/ubuntu-worker/tasks/rebar.yml

```diff
@@ -0,0 +1,37 @@
+- name: create rebar group
+  group:
+    name: rebar
+    state: present
+
+- name: create rebar user
+  user:
+    name: rebar
+    createhome: yes
+    group: rebar
+    state: present
+
+- name: download rebar
+  get_url:
+    url: https://github.com/rebar/rebar/archive/{{ rebar_version }}.tar.gz
```

Comment:
That is need of old past days. Not saying that this is wrong and wont work, but if you can make things simpler - why not?(: